lib/standalone.mjs

```diff
@@ -111,7 +111,7 @@ export default async function standalone({
 		emitDeprecationWarning(
 			'"github" formatter is deprecated.',
 			'GITHUB_FORMATTER',
-			'Please use "stylelint-actions-formatters" instead.',
+			'Please use "@csstools/stylelint-formatter-github" instead.',
```

Comment:
[question]

Should we maybe link to the list here: https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters

Then users can pick the one they prefer and we won't need to update the message if a better formatter appears in the future.

Maybe something like:

```
See https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters for alternative formatters
```

Reply:
Good suggestion! 
Linking to the list makes it more maintainable.

Reply:
Fixed in this commit. 
[update message](https://github.com/stylelint/stylelint/pull/8115/commits/306b314cbd1e843feec346a8035c3063354e1e2c)

Reply:
Thank you!